roles/eda/tasks/deploy_eda.yml

```diff
@@ -1,4 +1,18 @@
 ---
+- name: Wait for {{ deployment_type }}restore to complete
+  kubernetes.core.k8s_info:
+    api_version: "{{ api_version }}"
+    kind: "{{ deployment_type }}restore"
```

Comment:
just to be sure, we're waiting for an `EDARestore` kind to be finished right ?

if so, does the uppercases matter ?

Reply:
Good callout. Thankfully, it looks like it doesn't matter.  

```
[chadams@awx eda-server-operator]$ oc get edarestore
NAME       AGE
restore1   5s
[chadams@awx eda-server-operator]$ oc get EDARestore
NAME       AGE
restore1   10s
[chadams@awx eda-server-operator]$ oc get EDArestore
NAME       AGE
restore1   14s
```